app/models/sac_cas/role/mitglied_family_validations.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2023, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas
+
+module SacCas::Role::MitgliedFamilyValidations
+  extend ActiveSupport::Concern
+
+  MAXIMUM_ADULT_FAMILY_MEMBERS_COUNT = 2
+
+  included do
+    validate :assert_adult_family_members_count
+    validate :assert_family_on_primary_group
+  end
+
+  private
+
+  # There can only be MAXIMUM_ADULT_FAMILY_MEMBERS_COUNT adults with beitragskategory=family in a household.
+  def assert_adult_family_members_count
+    return unless person&.household_key? &&
+      beitragskategorie == SacCas::Beitragskategorie::Calculator::CATEGORY_FAMILY.to_s &&
+      SacCas::Beitragskategorie::Calculator::AGE_ADULT.cover?(person.years)
+
+    adult_housemates_count = person.
```

Comment:
könnte man dies nicht in eine eigene oder gar mehrere methode auslagern?
einrücken?